docs/manual/core-concepts/getting-started.md

```diff
@@ -4,13 +4,13 @@ In this tutorial you will learn to make a simple setup of Sequelize.
 
 ## Installing
 
-Sequelize is available via [npm](https://www.npmjs.com/package/sequelize) (or [yarn](https://yarnpkg.com/package/sequelize)).
+Sequelize is available via [npm](https://www.npmjs.com/package/@sequelize/core) (or [yarn](https://yarnpkg.com/package/@sequelize/core)).
 
 ```sh
 # using npm
-npm i sequelize # This will install latest version of Sequelize
+npm i @sequelize/core # This will install latest version of Sequelize
```

Comment:
```suggestion
npm i sequelize # This will install v6, the latest stable release of Sequelize
npm i @sequelize/core # This will install v7, the latest alpha release of Sequelize
```